### PR TITLE
New map: Sea Battle

### PR DESCRIPTION
**Pull request by Emojigit**

### CODE REVIEW

**Comment by Thomas--S** [comment]
Seems like license information is missing.

**Comment by Emojigit** [comment]
Cc by sa 4.0, why I missed info, and how to add?

**Comment by Thomas--S** [comment]
You can add a line `license = ...` to the map.conf file.

**Comment by Emojigit** [comment]
Sloved

**Comment by PiMonkeyServers** [comment]
This map is being rejected due to it being very plain and also similar to another map already on the server called "big ocean"

**Comment by PiMonkeyServers** [comment]
Reviewing you map showed us a few issues and bugs
nothing too big 

- Fix blues spawn. its missing two logs so there are 2 torches floating
bug1
- Fix the border. there are several plants where the border is meaning there are holes, remove the plants where the border is to fix this
bug3
- There is an error when running your map due to the original map world having a butterfly in it when you exported it. please get rid of the butterfly in the world 
bug2
- And finally take a new screenshot once all those bugs are fixed

**Comment by Emojigit** [comment]
> Reviewing your map showed us a few issues and bugs
> nothing too big
> 
>     * Fix blues spawn. its missing two logs so there are 2 torches floating
>       (IMG)
> 
>     * Fix the border. there are several plants where the border is meaning there are holes, remove the plants where the border is to fix this
>       (IMG)
> 
>     * There is an error when running your map due to the original map world having a butterfly in it when you exported it. please get rid of the butterfly in the world
>       (IMG)
> 
>     * And finally take a new screenshot once all those bugs are fixed
OK, but I have some qustion:
 - why those logs disappers?
 - why CTF don't make some red block entity at places that unreplaceable?

**Comment by JostP** [comment]
> OK, but I have some qustion:
> 
>     * why those logs disappers?
I'm not sure maybe you used node that is not supported in CTF


>     * why CTF don't make some red block entity at places that unreplaceable?

Yeah, that would be great, but someone who know how to do this must write some code. Currently rubenwardy don't have time to develop CTF.

**Comment by JostP** [comment]
still work on this?

File: sea_battle/map.conf [hunk 1]
```diff
@@ -0,0 +1,13 @@
+team.2.pos = (102,-36,-103)
+license = CC BY-SA 4.0
+r = 115
+team.1.pos = (-98,-29,98)
+initial_stuff = default:pick_steel,default:torch 10
+team.2 = blue
+rotation = z
+name = Sea Battle
+author = Cato
```
Comment (Panquesito7) [inline sea_battle/map.conf:9]: Is this map by Cato or Emojigit (or are both of them the same person)?

**Comment by JostP** [comment]
Re-open when you fix the bugs

**Pull request closed without merging**